src/observer/json_rows.py

```diff
@@ -0,0 +1,28 @@
+from collections import OrderedDict
+import itertools
+from . import utils
+from django.http import StreamingHttpResponse
+
+def streaming_response(filename, rows):
+    body = []
+    body.append((utils.safe_json_dumps(row) + "\n") for row in rows)
+    response = StreamingHttpResponse(itertools.chain.from_iterable(body), content_type="application/json")
```

Comment:
since it's multiple lines, the whole response is not valid JSON hence `content_type` needs to be something else. Trying to find a valid value

Reply:
https://github.com/wardi/jsonlines/issues/9 says `application/x-ndjson` which is from a "competing" standard but better than inventing a new value. The Python library does support both "standards".